Ticket-booking-spring-app/src/main/java/com/example/demo/entity/Screening.java

```diff
@@ -0,0 +1,83 @@
+package com.example.demo.entity;
+
+import jakarta.persistence.*;
+import org.springframework.cglib.core.Local;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+@Entity
+public class Screening {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="id", nullable = false)
+    private int id;
+    @Column(name="movie_title", nullable = false)
+    private String movieTitle;
```

Comment:
Rozróżnianie filmów tylko po tytule? A jak będziemy chcieli sprawdzić wszystkie rezerwacje dla Znachora z 2023 roku, a przy okazji nie wziąć rezerwacji do filmu z 1981 roku? ;)

Reply:
Można by stworzyć dodatkową tabelę "movie" z id, reżyserem itp, ale znowu zdecydowałam się uprościć, ponieważ do zrealizowania założeń zadania nie wydawało mi się to konieczne.